crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2148,6 +2253,24 @@ impl SymbolicationActor {
                 stacktraces,
                 options,
             };
+
+            // Save the minidump if there was a stackwalking difference or the new stackwalking method performed poorly.
+            if let Some(problem) = new_stackwalking_problem {
+                let msg = match problem {
+                    NewStackwalkingProblem::Diff(_) => "Different stackwalking results",
+                    NewStackwalkingProblem::Slow => "Slow stackwalking run",
+                };
+                sentry::with_scope(
+                    |scope| {
+                        if let NewStackwalkingProblem::Diff(diff) = problem {
+                            scope.set_extra("diff", sentry::protocol::Value::String(diff));
+                        }
+                    },
+                    || self.maybe_persist_minidump(minidump_file),
+                );
+                sentry::capture_message(msg, sentry::Level::Error);
```

Comment:
This `capture_message` needs to be inside the `with_scope` closure, otherwise the event will not have the `diff` attached.

Reply:
this :)